spec/book_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './../book'
+
+describe 'Book class' do
+  context 'Create a  book when the class is executed' do
+    it 'should create a new book' do
+      book = Book.new('Beautiful mamee', 'David')
+      expect(book.title).to eq 'Beautiful mamee'
+      expect(book.author).to eq 'David'
+    end
+  end
+end
```

Comment:
Since the book class has the `add_rental` method. Kindly test it using the Book class as well. i.e `rental = book.add_rental(person, date)`.

- Since the  `add_rental` method has `self` as a parameter. The book you created will add itself. Then you should `expect(book.rentals).to eql [rental]`

Remember that it is part of the requirements to test all methods in each classs:
![image](https://user-images.githubusercontent.com/38088818/201340227-da23e20f-028f-4033-9ccb-8e92b4f5639d.png)